Content.Shared/_RMC14/Damage/RMCTemporaryInvincibilityComponent.cs

```diff
@@ -0,0 +1,11 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._RMC14.Damage;
+
+
+[RegisterComponent, NetworkedComponent]
```

Comment:
Missing this so the field auto network does anything
```suggestion
[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
```